homeassistant/components/system_bridge/sensor.py

```diff
@@ -284,15 +281,17 @@ async def async_setup_entry(
             )
 
     displays = []
-    for display in coordinator.data.display.displays:
-        displays.append(
-            {
-                "key": display,
-                "name": getattr(coordinator.data.display, f"{display}_name").replace(
-                    "Display ", ""
-                ),
-            },
-        )
+    if coordinator.data.display.displays is not None:
+        for display in coordinator.data.display.displays:
+            if hasattr(coordinator.data.display, f"{display}_name"):
+                displays.append(
+                    {
+                        "key": display,
+                        "name": getattr(
+                            coordinator.data.display, f"{display}_name", ""
+                        ).replace("Display ", ""),
+                    },
+                )
```

Comment:
You can use a list comprehension for this, resulting in a single `extend` call.
```suggestion
        displays.extend(
            {
                "key": display,
                "name": getattr(
                    coordinator.data.display, f"{display}_name", ""
                ).replace("Display ", ""),
             }
             for display in coordinator.data.display.displays
             if hasattr(coordinator.data.display, f"{display}_name")
        )
```